code/game/objects/items/kitchen.dm

```diff
@@ -92,8 +92,12 @@
 	var/bayonet = FALSE	//Can this be attached to a gun?
 	custom_price = 250
 
-/obj/item/kitchen/knife/Initialize()
+/obj/item/kitchen/knife/ComponentInitialize()
 	. = ..()
+	set_butchering()

```

Comment:
It would be ideal if this was moved to `/obj/item` but that would mean touching on several other items with the butchering component.
If you are unwilling to do this, it's fine. If you do, it'd be very appreciated.

Reply:
Actually, forget that. Talked with ninjanomnom and that is not really necessary. What I asked was a bit unnecessary, sorry for that.
We can leave it here as is.